### ISSUE TITLE
Not able to install extra

### ISSUE THREAD

**Issue by yoli1888** [nidhaloff/deep-translator#191]
* deep_translator version: 1.10.0
* Python version: 3.10
* Operating System: Linux

### Description

Attempt to install deep-translator extras after installing deep-translator 1.10.0:  [docx] and [pdf]

### What I Did

> pip install -U deep-translator[pdf]
zsh: no matches found: deep-translator[pdf]

**Comment by yoli1888** [comment]
Turns out zsh uses square brackets for globbing / pattern matching.

So I try again with: 

> pip install 'deep-translator[pdf]'
WARNING: deep-translator 1.10.0 does not provide the extra 'pdf'

Appreciate if python examples can be provided to verify if deep-translator can process pdf/docx file.

**Comment by nidhaloff** [comment]
Hey @yoli1888, thanks for opening this issue. Indeed it was a bug in the way I was using poetry. I released a hotfix `v1.10.1`

So if you want to use extras, you have to install options:

```
- pip example -> pip install deep-translator[pdf]
- poetry example -> poetry install deep-translator --extras "pdf"
```
